app/assets/error_pages/5xx.html

```diff
@@ -125,21 +125,13 @@ <h1 class="heading-large">
 
     <script src="/static/javascripts/govuk-template.js"></script>
 
-  <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-    ga('create', 'UA-75215134-1', 'auto');
-    ga('set', 'anonymizeIp', true);
-    ga('set', 'displayFeaturesTask', null);
-    ga('set', 'transport', 'beacon');
-    // strip UUIDs
-    page = (window.location.pathname + window.location.search).replace(
-      /[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}/g, '…'
-    )
-    ga('send', 'pageview');
-  </script>
-
+    <script>
+      if (window.GOVUK.cookie('_ga') !== null) {
+        document.cookie = '_ga' + '=;expires=' + new Date() + ';domain=.' + window.location.hostname.replace(/^www/, '') + ';path=/';
```

Comment:
This is going to result in `";domain=..notifications.service.gov.uk"` – not sure if that will work.

This will work for www, non-www, and localhost:
```suggestion
        document.cookie = '_ga' + '=;expires=' + new Date() + ';domain=' + window.location.hostname.replace(/^www\./, '.') + ';path=/';
```

Reply:
Rats, good point